mambaTests/Fixtures/hls_singleMediaFile.txt

```diff
@@ -1,5 +1,6 @@
 #EXTM3U
 #EXT-X-TARGETDURATION:10
+#EXT-X-BITRATE:3000000
```

Comment:
If the unit tests pass, this is OK. In general it's bad to mess around with existing fixtures since they are potentially used by multiple unit tests, and frequently the number of lines/relative positions of tags are part of the test.

Reply:
That's a good point, thanks.